specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -0,0 +1,1045 @@
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/ResourceNameInParameters"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Quota information will be returned as response with pagination.",
+						"schema": {
+							"$ref": "#/definitions/CurrentQuotaLimit"
+						},
+						"headers": {
+							"ETag": {
+								"description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+								"type": "string"
+							}
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				}
+			},
+			"put": {
+				"tags": [
+					"QuotaRequests"
+				],
+				"summary": "Submits a Quota Request for a resource provider at the specified location for the specific resource in the parameter.",
+				"description": "Submits Quota change request for a resource provider for the specified location for the specific resource in the parameter. To use, first make a Get request to get quota information. This information consists of a list of resources and information regarding those resources. For all the resources in that list which require an update to their quotas, update their limit fields in the response from the Get request to their new values. Then, submit this updated JSON object to this quota request API. This will update the quotas to the values specified.",
+				"operationId": "QuotaRequest_Create",
+				"x-ms-examples": {
+					"Quotas_Request_listForCompute": {
+						"$ref": "./examples/putComputeOneSkuQuotaRequest.json"
+					}
+				},
+				"x-ms-long-running-operation": true,
+				"x-ms-long-running-operation-options": {
+					"final-state-via": "location"
+				},
+				"parameters": [
+					{
+						"$ref": "#/parameters/SubscriptionIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ProviderIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ResourceNameInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"name": "createQuotaRequest",
+						"in": "body",
+						"description": "Quota requests payload.",
+						"required": true,
+						"schema": {
+							"$ref": "#/definitions/CurrentQuotaLimit"
+						}
+					},
+					{
+						"$ref": "#/parameters/IfMatchRequiredParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Returns Quota requests details.",
+						"schema": {
+							"$ref": "#/definitions/QuotaRequestDetails"
+						}
+					},
+					"202": {
+						"description": "The request is accepted and is being processed. Please use requestId to check the status.",
+						"schema": {
```

Comment:
this should not have any response body. The location header to track the status of this operation is provided as a header.

Reply:
Changed it to 201 response as @KrisBash suggested.